### PR TITLE
replace makestyles from @mui/styles by tss react

### PR DESCRIPTION
**Pull request by Med1233**
## Description
replace makestyles from @mui/styles by tss react

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x (master)

<h2> How this pull request can be tested ? </h2>

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "build:dev": "cross-env WEBPACK_ENV=development webpack serve --mode development --config ./webpack.config.dev.js",
     "eslint": "eslint ./www/front_src --ext .js,.jsx,.ts,.tsx",
     "eslint:fix": "npm run eslint -- --fix",
-    "test": "jest",
+    "test": "jest --silent",
```
Comment (JKancel) [inline package.json]: Is it needed ?

Comment (Med1233) [reply package.json]: No. this is just to prevent logging warnings in the test command. I will undo this when I'm done

Comment (Med1233) [reply package.json]: By the way, this is so helpful for detecting failed tests quickly. Should we consider adding a new command for this? what do you think  @Thebarda ?

Comment (Thebarda) [inline package.json:11]:
```suggestion
    "test": "jest",
```

Comment (Thebarda) [reply package.json]: Yes it's better to create another command jest:silent

**Pull request closed without merging**